src/YesSql.Core/Provider/IDbProviderOptions.cs

```diff
@@ -0,0 +1,11 @@
+﻿using YesSql.Core.Services;
+
+namespace YesSql.Core.Provider
+{
+    public interface IDbProviderOptions
+    {
+        string ProviderName { get; set; }
```

Comment:
Ok, last thing, if I understand you correctly `UseXXX` without connection string is Orchard CMS specific thing, so should be added there